README.md

```diff
@@ -111,6 +111,9 @@ The following dependencies should be ignored:
 - Vue Test Utils (`@vue/test-utils`):
   Seems to require a newer Vue.js version (see [#492](https://github.com/wmde/new-lexeme-special-page/pull/492)),
   though it’s worth checking whether that’s still the case in newer VTU versions.
+- lint-staged:
+  [v15.0.0](https://github.com/lint-staged/lint-staged/releases/tag/v15.0.0 drops compatibility with Node 16,
+  which we still use in CI (see also “Prerequisites” above).
```

Comment:
Mh, why are we still using that in CI? As I understand it, Node 18 and Node 20 images already exist: [⚓ T331181 Create WMF CI images and jobs for Node.js 18](https://phabricator.wikimedia.org/T331181). We should probably at least have a (maybe stalled) task to upgrade this Node 18 or maybe directly to Node 20. And mention/link the task here.

Reply:
Ah, this would probably actually be waiting on [⚓ T331180 Upgrade all CI jobs for WMF-deployed projects from Node 16 to Node 18](https://phabricator.wikimedia.org/T331180) instead.

Reply:
Yeah, the images exist (Fresh also uses them) but they’re not used in CI yet.

I guess `lint-staged` in particular doesn’t need to run in CI, but if we upgrade it then we get into a weird situation where you need Node 18 to develop locally, but also the build still has to stay compatible with Node 16 (so we can check the dist diff in CI)…

Reply:
Makes sense. Then linking T331180 in this section would be good enough for me. Then we can easily check during future chores if we can move forward with this or still have to wait.

Reply:
Done.